background.js

```diff
@@ -202,3 +202,25 @@ chrome.storage.local.get('version', prefs => {
 		}
 	}
 });
+
+chrome.runtime.onInstalled.addListener(({reason, previousVersion, id} = {}) => {
+	// reason: install, update, chrome_update, shared_module_update
+	if (!/update/.test(reason)) {
+		return;
+  }
+  console.log(reason);
+	const contentScripts = chrome.app.getDetails().content_scripts;
+	chrome.windows.getAll({populate: true}, windows => {
+		windows.forEach(win => {
+			win.tabs.forEach(tab => {
```

Comment:
Instead of having two loops, why not just have something like this
```js
chrome.tabs.query({url: ['*://*/*']}, tabs => ...)
```
This will ignore `chrome://` URLs as well